tests/utils/features/query_features.py

```diff
@@ -406,20 +565,53 @@ def add_list_parser(subparsers: argparse._SubParsersAction):
     cmd = 'list'
     lst: argparse.ArgumentParser = subparsers.add_parser(cmd, help='lists all timestamps with systems present in data source',
                                 description='Lists all timestamps with systems present in data source.')
-    lst.add_argument('-f', '--file', help='json file containing creds for mongodb', type=argparse.FileType('r', encoding='utf-8'))
-    lst.add_argument('-d', '--dir', help='folder containing feature data', type=str)
+    add_data_source_args(lst)
     return cmd
 
 
+def add_all_features_parser(subparsers: argparse._SubParsersAction):
```

Comment:
spotted a couple other places missing `-> None`, please have a look at the diff.

```suggestion
def add_all_features_parser(subparsers: argparse._SubParsersAction) -> None:
```